oci/client/list.go

```diff
@@ -118,3 +121,12 @@ func (c *Client) List(ctx context.Context, url string, opts ListOptions) ([]Meta
 
 	return metas, nil
 }
+
+func isCosignArtifact(tag string) bool {
```

Comment:
Let’s make this public as we need it in image-reflector-controller, also please add a comment to it.